test/router/dnssec/dnssec_suite_test.go

```diff
@@ -20,14 +20,14 @@ package dnssec_test
  */
 
 import (
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
+	// _ "github.com/onsi/ginkgo"
+	// _ "github.com/onsi/gomega"
```

Comment:
These can just be removed, right?

Reply:
Actually, based on what I said below, I think I should change those back to `.` imports. `.` and `_` don't do the same thing for imports, which is what I thought when I changed that. When you use `.` it imports all symbols from the package into the current namespace, which might be necessary for the code below to compile (although it actually didn't compile). So if I'm leaving in the below code, I'll wanna fix this to just be commented-out and not also changed.